relayer/config-testnet.toml

```diff
@@ -120,13 +120,13 @@ trust_threshold = { numerator = '1', denominator = '3' }
 memo_prefix = 'agoric-emerynet-relay'
 
 [[chains]]
-id = 'agoricdev-23'
-rpc_addr = 'https://devnet.rpc.agoric.net'
-grpc_addr = 'https://devnet.grpc.agoric.net'
-event_source = { mode = 'push', url = 'wss://devnet.rpc.agoric.net:26657/websocket', batch_delay = '500ms' }
```

Comment:
just making sure: protocol changed from 'wss' to 'ws'. is this a typo?

Reply:
`wss` connects over HTTPS, while `ws` connects over HTTP. Since we are using a local chain, we have switched to `ws`.